src/answers/services/answers.service.ts

```diff
@@ -49,7 +49,7 @@ export class AnswersService {
     user: IUser;
   }): Promise<IAnswer[]> {
     let filter = {};
-    const confirmedUsers = [];
+    const confirmedUsers: (mongoose.Types.ObjectId | string | undefined)[] = [];
```

Comment:
```suggestion
    const confirmedUsers: (mongoose.Types.ObjectId)[] = [];
```

Reply:
This needs to be such a type because line 54 adds a string type and line 64 adds an undefined type

Reply:
Line 54 should be an ObjectId, not a string. Do we need undefined? I thought types only applied at compilation but not at runtime